src/styles/blocks/calendar.scss

```
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-day-gap;
+  padding: $calendar-padding;
+  min-width: getWidth(
+    $calendar-day-size,
+    $calendar-columns,
+    $calendar-day-gap
+  );
+  max-width: getWidth(
+    $calendar-day-size,
+    $calendar-columns,
+    $calendar-day-gap
+  );
```

why do you have both of them? you literally just can replace them with `width`
```suggestion
  width: getWidth(
    $calendar-day-size,
    $calendar-columns,
    $calendar-day-gap
  );
```